app/components/provider_interface/conditions_component.html.erb

```diff
@@ -11,13 +11,7 @@
           <tr class="govuk-table__row conditions-row">
             <td class="govuk-table__cell"><%= condition.text %></td>
             <td class="govuk-table__cell govuk-table__cell--numeric">
-              <% if condition.met? %>
-                <%= govuk_tag(text: 'Met', colour: 'green') %>
-              <% elsif condition.unmet? %>
-                <%= govuk_tag(text: 'Not met', colour: 'red') %>
-              <% else %>
-                <%= govuk_tag(text: 'Pending', colour: 'grey') %>
-              <% end %>
+              <%= render ProviderInterface::ConditionStatusTagComponent.new(condition) %>
```

Comment:
These places should really be using the same conditions list component. It's a component that's needed in the next phase of the work too, so I'll add it in this PR too

Reply:
We can address this after the individual conditions work is done. I'll add a component to handle this and we can refactor old code to use the new component